docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
```

Comment:
Any particular reason why you took the `.devel` off of this? It was there to make it explicitly clear that this isn't a production ready Docker Compose configuration.

Reply:
To improve the developer experience. 
```
docker-compose -f docker-compose.devel.yml up
```

vs.
```
docker-compose up
```

I can add a comment to the readme to clarify that it's not a production ready instance. Although at this point, adding some parameterisation and it could be production ready for micro instances.

Reply:
:thinking: @robmarch2 @garyttierney thoughts?

Reply:
I don't have a particularly strong opinion in this instance seeing as I only ever tend to run the application via gradle. In general I tend to come down on the side of being explicit, but in this case if this is the same `docker-compose.yml` that will be used when it is prod ready, I don't see much point in temporarily having the `.devel`. If on the other hand, the intention is to have separate ones for local development vs prod, it should stay.